libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -633,7 +654,7 @@ interface BlockingBreezServices {
    RecommendedFees recommended_fees();
 
    [Throws=SdkError]
-   BuyBitcoinResponse buy_bitcoin(BuyBitcoinRequest req);
+   BuyBitcoinResponse buy_bitcoin(BuyBitcoinRequest request);
```

Comment:
NIT: IMO `req` and `req_data` is better because it's less verbose (causes shorter lines of code, so less linebreaks when formatting), while the name is still clear enough as to what it means.

Reply:
We were using `request` in some places. It's all good as long as they are consistent

Reply:
Done e48902088783449e148bba6c6b89528c0fa21451